configs/sot/siamese_rpn/siamese_rpn_r50_20e_lasot.py

```diff
@@ -103,47 +108,38 @@
 # dataset settings
 data = dict(
     samples_per_gpu=28,
-    workers_per_gpu=2,
-    train=[
-        dict(
-            type='RepeatDataset',
-            times=39,
-            dataset=dict(
-                type='SOTTrainDataset',
+    workers_per_gpu=8,
```

Comment:
Suggest to set it to 4 in the config, since the user's machine may not support so many workers.